lib/spack/spack/build_systems/meson.py

```diff
@@ -116,6 +116,11 @@ def _std_args(pkg):
 
         args = [
             '--prefix={0}'.format(pkg.prefix),
+            # If we do not specify libdir explicitly, Meson chooses something
+            # like lib/x86_64-linux-gnu, which causes problems when trying to
+            # find libraries and pkg-config files.
+            # See https://github.com/mesonbuild/meson/issues/2197
+            '--libdir={0}'.format(pkg.prefix.lib),
```

Comment:
should this be something like `.libs[0]` or will it always be `lib` (rather than `lib64`)?

Reply:
This is where we want Meson to install the libraries, so it is completely up to us. I checked a few other (autotools) packages and they also installed into lib instead of lib64.